lib/src/widgets/chatui_textfield.dart

```diff
@@ -36,48 +36,121 @@ class ChatUITextField extends StatelessWidget {
   final FocusNode focusNode;
   final TextEditingController textEditingController;
   final VoidCallBack onPressed;
+
+  @override
+  State<ChatUITextField> createState() => _ChatUITextFieldState();
+}
+
+class _ChatUITextFieldState extends State<ChatUITextField> {
+  String _inputText = '';
+
+  final ImagePicker _imagePicker = ImagePicker();
+
+  SendMessageConfiguration? get sendMessageConfig => widget.sendMessageConfig;
+
+  ImagePickerIconsConfiguration? get imagePickerIconsConfig =>
+      sendMessageConfig?.imagePickerIconsConfig;
   final _outLineBorder = OutlineInputBorder(
     borderSide: const BorderSide(color: Colors.transparent),
     borderRadius: BorderRadius.circular(27),
   );
 
   @override
   Widget build(BuildContext context) {
-    return TextField(
-      focusNode: focusNode,
-      controller: textEditingController,
-      style:
-          sendMessageConfig?.textStyle ?? const TextStyle(color: Colors.white),
-      maxLines: 5,
-      minLines: 1,
-      textCapitalization: TextCapitalization.sentences,
-      decoration: InputDecoration(
-        suffixIcon: IconButton(
-          color: sendMessageConfig?.defaultSendButtonColor ?? Colors.green,
-          onPressed: onPressed,
-          icon: sendMessageConfig?.sendButtonIcon ?? const Icon(Icons.send),
-        ),
-        hintText: sendMessageConfig?.hintText ?? PackageStrings.message,
-        fillColor: sendMessageConfig?.textFieldBackgroundColor ?? Colors.white,
-        filled: true,
-        hintStyle: sendMessageConfig?.hintStyle ??
-            TextStyle(
-              fontSize: 16,
-              fontWeight: FontWeight.w400,
-              color: Colors.grey.shade600,
-              letterSpacing: 0.25,
+    return Container(
+      padding: const EdgeInsets.symmetric(horizontal: 6),
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(27),
```

Comment:
try to remove all the static values and provide user specific value, make sure it has default values

Reply:
Updated